ReplayMessenger.device.lib.nut

```diff
@@ -93,6 +109,51 @@ class ReplayMessenger {
         return _mm.send(messageName, data, null, _retryInterval, metadata);
     }
 
+    function beforeReplay(loggerName, callback = null) {
```

Comment:
It's better to extract common code for `beforeReplay` and `afterReplay` into a separate function.